utils/openocd/Makefile

```diff
@@ -34,7 +37,18 @@ define Package/openocd
   CATEGORY:=Utilities
   TITLE:=OpenOCD Utility
   URL:=http://openocd.sf.net/
-  DEPENDS:=+libusb-1.0 +libusb-compat +libftdi1 +hidapi
+  DEPENDS:=+PACKAGE_openocd_with_usb:libusb-1.0 \
+	+PACKAGE_openocd_with_usb:libusb-compat \
```

Comment:
really? Before it would autoinclude based on the usb packages I had selected, now I _have_to have libusb-compat?

Reply:
Before it would automatically pull in all the USB dependencies in any case. Now you can disable it. libusb-compat is really tiny too, it's a small wrapper around libusb.

Reply:
doh, sorry. I tried to check what had been removed, and completely missed the exact line above where it was pulling them in. :( I thought it had been relying on what had been otherwise selected and letting it autodetect from there.  Sure, this is certainly better than before :)